Sources/Amplitude/Storages/PersistentStorage.swift

```diff
@@ -91,6 +91,7 @@ class PersistentStorage: Storage {
                 return readV1File(content: content!)
             }
         } catch {
+            diagonostics.addErrorLog(error.localizedDescription)
```

Comment:
Is there a non localized version of the error that we can collect?

Reply:
Not that I can find. Localized here will be the default development language, not the device language. Alternatively we could log raw posix error codes, but it's not guaranteed that errors here will always be POSIX errors.